internal/utils/dataplane/config.go

```diff
@@ -57,7 +27,7 @@ var KongDefaults = map[string]string{
 // PodTemplateSpec.
 // EnvVars are sorted lexographically as a side effect.
 // It also returns the updated EnvVar slice.
-func FillDataPlaneProxyContainerEnvs(existing []corev1.EnvVar, podTemplateSpec *corev1.PodTemplateSpec) {
+func FillDataPlaneProxyContainerEnvs(existing []corev1.EnvVar, podTemplateSpec *corev1.PodTemplateSpec, envSet map[string]string) {
```

Comment:
Now that we're passing the third argument I believe we should also add a mention of it to the godoc. WDYT?